### ISSUE TITLE
bad accessibility

### ISSUE THREAD

**Issue by ahmedSamy31** [cmderdev/cmder#2334]
### Purpose of the issue
- [ ] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question

**Comment by daxgames** [comment]
@ahmedSamy31 I am sorry to hear about the difficulties.  Unfortunately there is nothing we, the Cmder Team, can do to help you.

The GUI/Terminal emulator we use is from a separate project called Conemu.  If there are requests to enable some accessibility features they should be made [here](https://github.com/Maximus5/ConEmu/issues)
